lib/node_modules/@stdlib/lapack/base/dge-trans/README.md

```diff
@@ -0,0 +1,243 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dge-trans
+
+> Convert input general matrix from row-major to column-major layout or vice versa.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var transpose = require( '@stdlib/lapack/base/dge-trans' );
```

Comment:
```suggestion
var dge-trans = require( '@stdlib/lapack/base/dge-trans' );
```

Shall I do the following or `transpose` is fine?

Reply:
You cannot go `dge-trans`. Do `dgetrans`?

Reply:
Done :)

Reply:
@Pranavchiku No. The package name was fine and as it should be `dge-trans`. The variable name cannot have a hyphen.

Reply:
I mean, we are also free to name things as we want. But my comment was specifically directed to `var dge-trans`, which is not valid JS.

Reply:
So shall I revert back to package `dge-trans`?

Reply:
If you don't mind. 😬